frontend/src/Pages/Teacher/Course/TeacherCourseResources.jsx

```
@@ -0,0 +1,243 @@
+                                )}
+                                <p className="text-gray-500 text-sm mt-1">
+                                    <strong>Type:</strong> {resource.file_type} |{" "}
+                                    <strong>Size:</strong> {(resource.file_size / 1024).toFixed(2)} KB
+                                </p>
+                            </div>
+
+                            <div className="flex gap-2 mt-4 sm:mt-0">
+                                <button
+                                    onClick={() => handleDownload(resource.id, resource.file_name)}
+                                    className="bg-green-600 hover:bg-green-700 text-white px-3 py-2 rounded-md text-sm flex items-center gap-1"
+                                >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)